ci/run-sanity.sh

```diff
@@ -39,5 +52,15 @@ $solana_ledger_tool create-snapshot --ledger config/ledger "$snapshot_slot" conf
 cp config/ledger/genesis.tar.bz2 config/snapshot-ledger
 $solana_ledger_tool copy --ledger config/ledger \
   --target-db config/snapshot-ledger --starting-slot "$snapshot_slot" --ending-slot "$latest_slot"
-$solana_ledger_tool verify --ledger config/snapshot-ledger --block-verification-method blockstore-processor
-$solana_ledger_tool verify --ledger config/snapshot-ledger --block-verification-method unified-scheduler
+$solana_ledger_tool verify --abort-on-invalid-block \
+  --ledger config/snapshot-ledger --block-verification-method blockstore-processor
+$solana_ledger_tool verify --abort-on-invalid-block \
+  --ledger config/snapshot-ledger --block-verification-method unified-scheduler
+
+first_simulated_slot=$((latest_slot / 2))
+echo "First simulated slot: ${first_simulated_slot}"
+touch config/ledger/simulate_block_production_allowed
+$solana_ledger_tool simulate-block-production --ledger config/ledger \
+  --first-simulated-slot $first_simulated_slot
+$solana_ledger_tool verify --abort-on-invalid-block \
+  --ledger config/ledger --enable-hash-overrides
```

Comment:
this is all the test this new debug functionality has... I think it's okay. but open to suggestions to add some unit tests, if wanted.